### DISCUSSION TITLE
Calculating nematic order parameter from GSD file

### DISCUSSION THREAD

**Discussion by annaashki** [glotzerlab/freud#1197] [Support] [answered]
basic query - RESOLVED

**Comment by joaander** [comment, marked as answer]
Use `mode='r'`.

This is documented in the [GSD changelog](https://gsd.readthedocs.io/en/v3.2.0/changes.html#id5) and [documentation for open](https://gsd.readthedocs.io/en/v3.2.0/python-module-gsd.hoomd.html#gsd.hoomd.open).

**Reply by annaashki** [reply]
Thank you, this has worked -- the other errors remain unchanged so the type wasn't causing them. Do you have any idea what might be?

**Reply by tommy-waltmann** [reply]
The `compute` method of the `Nematic` class takes one argument which is the orientations, it does not take a trajectory frame. This is stated in the [documentation](https://freud.readthedocs.io/en/latest/modules/order.html#freud.order.Nematic)

**Reply by annaashki** [reply]
Fixed now, thank you!
